### ISSUE TITLE
[notice] https://github.com/eyalroz/printf  <-This repo is maintaining

### ISSUE THREAD

**Issue by mysterywolf** [mpaland/printf#128]
https://github.com/eyalroz/printf

**Comment by laoshaw** [comment]
Maybe the two repos can be merged or sync-ed somehow?

**Comment by eyalroz** [comment]
@laoshaw : I would be glad to coordinate with mpaland, but he has been unavailable for a long period of time. By now, there are significant differences throughout the library code; and this library is not seeing any development in other directions.

Bottom line: Just use [eyalroz/printf](https://github.com/eyalroz/printf), there's nothing here you would miss.

**Comment by laoshaw** [comment]
Thanks. I will switch to eyalroz/printf for now then.

**Comment by bwxz** [comment]
改了不少东西啊，宏定义开关也改了

**Comment by eyalroz** [comment]
Indeed, many things have been changed - to fix bugs, and address the needs of more usage scenarios.

**Comment by ghost** [comment]
> https://github.com/eyalroz/printf

Fork hasn't been touched it looks like. Oh well

**Comment by eyalroz** [comment]
> > https://github.com/eyalroz/printf
> 
> Fork hasn't been touched it looks like. Oh well

The fork is being maintained, my friend.

**Comment by ghost** [comment]
> > > https://github.com/eyalroz/printf
> > 
> > 
> > Fork hasn't been touched it looks like. Oh well
> 
> The fork is being maintained, my friend.

Oh, okay! My apologies
